eng/doc/fips/UserGuide.md

```diff
@@ -645,6 +658,204 @@ sha512.Sum512_224 is not FIPS compliant.
 
 sha512.Sum512_256 is not FIPS compliant.
 
+### [crypto/rsa](https://pkg.go.dev/crypto/rsa)
+
+Package rsa implements RSA encryption as specified in PKCS #1 and RFC 8017.
+
+#### func [DecryptOAEP](https://pkg.go.dev/crypto/rsa#DecryptOAEP)
+
+```go
+func rsa.DecryptOAEP(h hash.Hash, rand io.Reader, priv *rsa.PrivateKey, ciphertext []byte, label []byte) ([]byte, error)
+```
+
+DecryptOAEP decrypts ciphertext using RSA-OAEP.
+
+**Parameters**
+
+`h` must be the result of one of the following functions in order to be FIPS compliant: sha1.New(), sha224.New(), sha256.New(), sha384.New(), or sha512.New().
+If this invariant is not met, DecryptOAEP won't be FIPS compliant but still will decrypt the message.
+
+`rand` is not used.

```

Comment:
OpenSSL RSA blinging is opt-out since 0.9.6. This fact is not documented in the official docs, I've only seen a comment here: https://github.com/openssl/openssl/blob/12ad22dd16ffe47f8cde3cddb84a160e8cdb3e30/crypto/rsa/rsa.h#L204-L210, which matches what's implemented.

I'll extend the `rand` comment noting that blinding does not depend on this parameter.